__tests__/app.test.js

```diff
@@ -30,19 +30,73 @@ describe("API Testing", () => {
   });
 
   describe("Ticket 4: GET /api/articles", () => {
-    test("returns an array of articles which is sorted by date in descending order ", () => {
+    test("returns an array of articles when passing query of valid sorted_by", () => {
       return request(app)
-        .get("/api/articles")
+        .get("/api/articles?sort_by=title")
         .expect(200)
         .then(({ body }) => {
-          const objOfArr = body.articles;
-          expect(objOfArr.length).toBe(12);
-          expect(objOfArr[0].created_at).toBe("2020-11-03T09:12:00.000Z");
-          expect(objOfArr[objOfArr.length - 1].created_at).toBe(
-            "2020-01-07T14:08:00.000Z"
-          );
+          expect(body.articles.length).toBe(12);
+          expect(body.articles).toBeSorted({ descending: true });
+        });
+    });
+
+    test("returns an array of articles when passing query of valid order", () => {
+      return request(app)
+        .get("/api/articles?order=ASC")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles.length).toBe(12);
+          expect(body.articles).toBeSorted({ descending: false });
+        });
+    });
+
+    test("returns an array of articles when passing query of valid topic", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeSorted({ descending: false });
+          expect(body.articles.length).toBe(11);
+        });
+    });
+
+    test("returns an array of articles when passing non-existing topic", () => {
+      return request(app)
+        .get("/api/articles?howareyou=imfine")
```

Comment:
This should instead be a 400 bad request, we're sending through invalid info to make our request